js/index.js

```diff
@@ -42,28 +43,117 @@ function renderPepperoni() {
   });
 }
 
-function renderMushrooms() {
-  // Iteration 1: set the visibility of `<section class="mushroom">`
-}
+
 
 function renderGreenPeppers() {
-  // Iteration 1: set the visibility of `<section class="green-pepper">`
-}
+  
+    document.querySelectorAll('.green-pepper').forEach(greenPep => {
+      if (state.greenPeppers) {
+        greenPep.style.visibility = 'visible';
+      } else {
+        greenPep.style.visibility = 'hidden';
+      }
+    });
+  }
+  
+
+  function renderMushrooms() {
+  
+    document.querySelectorAll('.mushroom').forEach(mush=> {
+      if (state.mushrooms) {
+        mush.style.visibility = 'visible';
+      } else {
+        mush.style.visibility = 'hidden';
+      }
+    });
+  }
+  
+
 
 function renderWhiteSauce() {
   // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+
+  document.querySelectorAll('.sauce').forEach(sauce=> {
+    if (state.whiteSauce) {
+      sauce.classList.add('sauce-white');
+    } else {
+      sauce.classList.remove('sauce-white');
+    }
+  });
```

Comment:
as there is only one element with the class ``.sauce`` you don't need to use the ```querySelectorAll``` method
```
  let sauce = document.querySelector(".sauce");
  if (state.whiteSauce) {
    sauce.classList.add("sauce-white");
  } else {
    sauce.classList.remove("sauce-white");
  }
```